eduNEXT/xblock-mindmap
# feat: add translations in configs of the xblock

### Description
This PR adds translations to the configuration of the block. At the moment it only adds the translation to "Spanish (Latin America)" and "Spanish (Spain)".

### How To Test
1. Install this branch in your environment.
2. Change the platform language in `(Account → Site Preferences → Site Language)`. Set it to "Spanish (Latin America)"
3. Add a block in a unit from Studio.
4. You should see the block configuration in the selected language.

### Using translations commands
1. Run `make extract_translations`
2. Create a folder for the translations in `locale/`, ie: `locale/es_419/LC_MESSAGES/`, and create your `text.po` file.
3. Run `make compile_translations`

### Screenshots

1. Error in the LMS
![image](https://github.com/eduNEXT/xblock-mindmap/assets/64033729/6a12b956-ca6b-4245-bc94-420df44db69f)
2. Configuration edits in Studio
![image](https://github.com/eduNEXT/xblock-mindmap/assets/64033729/14523dd0-6f3c-49ea-91e8-1b491010b7b6)
3. The student_view in Studio
![image](https://github.com/eduNEXT/xblock-mindmap/assets/64033729/6ebb1004-1dc9-4829-b3c6-a3dbe84af907)
